stl/inc/limits

```diff
@@ -11,6 +11,8 @@
 #include <cfloat>
 #include <climits>
 #include <cwchar>
+#include <intrin.h>
+#include <type_traits>

```

Comment:
If all you need is `is_constant_evaluated()`, we should promote it to a more central header; I believe `<xtr1common>` would be appropriate.